src/plugins/files_management/public/app.tsx

```diff
@@ -58,6 +58,7 @@ export const App: FunctionComponent = () => {
               hits: files.map((file) => ({
                 id: file.id,
                 updatedAt: file.updated,
+                namespaces: file.namespaces ?? [],
```

Comment:
I think `namespaces` should always be present on the saved object, so we could also add an assertion operator rather than coercing to an empty array. WDYT?